### DISCUSSION TITLE
Automatic meal planner

### DISCUSSION THREAD

**Discussion by ga0202** [mealie-recipes/mealie#2785] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

It would be very nice if the meal planner fills itself automatically by the given rules or if there is a button to fill the whole planner with one click by the given rules.

### Please provide a concise description of the feature that would resolve your issue.

See above

### Please consider and list out some caveats or tradeoffs made in your design decision

-

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
Do the various "random" buttons on the meal planner already achieve this, on an individual day basis? And therefore this is effectively a request to do the same for all meals across all days in the defined date range?

**Reply by ga0202** [reply]
Yes and maybe an option that the meal planner fills itself automatically
